iati/default.py

```diff
@@ -69,7 +69,7 @@ def codelist(name, version=None):
 
     Args:
         name (str): The name of the Codelist to return.
-        version (str): The version of the Standard to return the Codelists for. Defaults to None. This means that the latest version of the specified Codelist is returned.
+        version (str): The version of the Standard to return the Codelists for. Defaults to None. This means that a version-independent version of the specified Codelist is returned.
```

Comment:
Documentation for the `version` arg needs sorting to be clearer and more correct.

Reply:
(across all docstrings)